src/components/InputForm.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import Styled from 'styled-components';
+import Button from './Button';
+
+const SearchGroup = Styled.form`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  width: 502px;
+  height: 32px;
+  margin-top: 20px;
+
+  &:before {
```

Comment:
you could use a simple div for this. Using a before here makes it a bit intransparent where the `r /` comes from when only looking at the JSX below